src/index.html

```diff
@@ -17,6 +30,77 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        class="logo"
+        src="images/logo.png"
+        alt="moyo-logo"
+      />
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav item">
+            <a href="#apple"class="nav-link is-active">Apple</a>
+          </li>
+          <li class="nav item">
+            <a href="#samsung"class="nav-link">Samsung</a>
+          </li>
+          <li class="nav item">
+            <a href="#smartphones"class="nav-link">Smartphones</a>
+          </li>
+          <li class="nav item"data-qa="hover"></li>
+          <a href="#lnC"class="nav-link">Laptops & Computers</a>
+          </li>
+          <li class="nav item"></li>
+          <a href="#gadgets"class="nav-link">Gadgets</a>
+          </li>
+          <li class="nav item">
+            <a href="#tablets"class="nav-link">Tablets</a>
+          </li>
+          <li class="nav item"></li>
+          <a href="#photo"class="nav-link">Photo</a>
+          </li>
+          <li class="nav item"></li>
```

Comment:
The `<li>` tag is not properly closed before the `<a>` tag starts. Ensure each `<li>` element properly wraps its content.